src/foam/u2/UnstyledTabs.js

```diff
@@ -41,14 +52,25 @@ foam.CLASS({
         this.tabRow.start('span').
           addClass(this.myClass('tab')).
           enableClass('selected', tab.selected$).
-          on('click', function() { this.selected = tab; }.bind(this)).
+          on('click', function() { 
+            this.selected = tab;
+            this.setMementoWithSelectedTab(tab);
+          }.bind(this)).
           add(tab.label$).
         end();
 
         tab.shown$ = tab.selected$;
       }
 
       this.SUPER(tab);
+    },
+    function setMementoWithSelectedTab() {
+      if ( ! this.updateMemento )
+        return;
+      if ( this.memento ) {
+        this.memento.paramsObj.selectedTab = this.selected.label;
+        this.memento.paramsObj = Object.assign({}, this.memento.paramsObj);
```

Comment:
is there a better way to do this?

Reply:
foam.Object.clone(this.memento.paramsObj) or foam.util.clone(this.memento.paramsObj) is easier to understand